src/quera_ahs_utils/plotting.py

```diff
@@ -248,3 +244,182 @@ def plot_avg_density(densities, register, with_labels = True, custom_axes = None
     else:
         return None,ax
 
+
+
+#%%
+def visualize_UDG(positions,
+                  radii,
+                  ax=None,
+                  label_nodes=False,
```

Comment:
`label_nodes` doesn't seem to be used for anything in visualize_UDG and should be removed (docstring should reflect this as well)

Reply:
No I think this is a bug, the flag should toggle the node labels, @jon-wurtz can you make sure this is working properly?